doc/Debugging.md

```diff
@@ -0,0 +1,61 @@
+# How to debug Conduktor platform 
+## Verify platform is up and running.
+
+For this example we'll use  [Conduktor's example-local](https://github.com/conduktor/conduktor-platform/tree/main/example-local).
+
+First step verify that all the components are running.
+```sh 
+mitch@m1-mbp example-local % docker-compose ps
+NAME                                 COMMAND                  SERVICE              STATUS              PORTS
+example-local-conduktor-platform-1   "/opt/conduktor/scri…"   conduktor-platform   running (healthy)   0.0.0.0:8080->8080/tcp
+example-local-kafka-1                "/opt/conduktor/scri…"   kafka                running             0.0.0.0:9092-9093->9092-9093/tcp, 0.0.0.0:9101->9101/tcp, 9999/tcp
+example-local-schema-registry-1      "/etc/confluent/dock…"   schema-registry      running             0.0.0.0:8081->8081/tcp
+example-local-zookeeper-1            "/docker-entrypoint.…"   zookeeper            running             2888/tcp, 3888/tcp, 0.0.0.0:2181->2181/tcp, 8080/tcp
+```
+
+If you are using an external Kafka installation you will only need to verify that the "conduktor-platform" container is showing "healthy" as the STATUS.
+
+If anything is not showing or showing "exited" as the status, a good first step is to check the `docker logs` output with `docker logs ${CONDUTKOR_PLATFORM_CONTAINER_NAME}`, for example in the example-local docker it would be `docker logs example-local-conduktor-platform-1`.  These logs can be saved to a file with `docker logs example-local-conduktor-platform-1 >& docker-logs-output.txt`.
```

Comment:
```suggestion
If anything is not showing or showing "exited" as the status, a good first step is to check the `docker logs` output with `docker logs ${CONDUKTOR_PLATFORM_CONTAINER_NAME}`, for example in the example-local docker it would be `docker logs example-local-conduktor-platform-1`.  These logs can be saved to a file with `docker logs example-local-conduktor-platform-1 >& docker-logs-output.txt`.
```